tests/test_resolutions.py

```diff
@@ -223,11 +250,13 @@ def test_descriptions(self) -> None:
             describe_mismatch="was not present",
             describe_none="was not present",
         )
+        ip = IsPresent()
 
-        _assert_descriptions(IsPresent(), element, expected)
+        assert ip.describe() == "present"
+        _assert_descriptions(ip.resolve(), element, expected)
 
     def test_type_hint(self) -> None:
         ip = IsPresent()
-        annotation = ip.__annotations__["matcher"]
+        annotation = ip.resolve.__annotations__["return"]
         assert annotation == "IsPresentElement"
-        assert type(ip.matcher) == IsPresentElement
+        assert type(ip.resolve()) == IsPresentElement
```

Comment:
Don't we have a `test_matches_protocols` test that we run? Maybe that's just for Actions?